components/list/list.js

```diff
@@ -0,0 +1,60 @@
+// Imports
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * General List component. Use when you need to display array of elements
+ */
+function List({ items, hideBullets, mods = [], align }) {
+  mods.push(`align_${align}`);
```

Comment:
just looked at the `master` branch. I see other components doing the same too. At some point they all need to be updated.

I remember dealing with a similar reference issue (by passing props in a loop) with @iwwwi last year.